docs/cimpl6Authoring.md

```diff
@@ -39,56 +37,56 @@ Model key:
 * Dark green: models created using CIMPL, leveraging OBF
 * Light yellow: mCode model, _constrains_ and _extends_ oncoCore, leverages OBF
 * Tan: models that _constrain_ and _extend_ mCode
-* Dark blue: FHIR resources and profiles
+* Dark blue: FHIR **resource**s and **profile**s
 
-CIMPL allows the modeling author to represent  FHIR profiles in one of two ways:
+CIMPL allows the modeling author to represent  FHIR **profile**s in one of two ways:
 
-* **Define the model specific resource attributes you need using FHIR resources or profiles as a base (*clean slate*).** In this approach, the modeling author already knows the FHIR resources or profiles to customize, and defines the element constraints or extensions in the new FHIR profile.
-* **Leverage CIMPL's _ObjectiveFHIR_ (OBF) base FHIR models** -  In this approach, the modeling author defines their FHIR profile and specifies a `Parent` class from the [ObjectiveFHIR User Guide](cimpl6ObjectiveFHIR.md) elements.
+* Define the model specific `Element`s you need using FHIR **resource**s or **profile**s as a base (_clean slate_). In this approach, the modeling author already knows the FHIR **resource**s or **profile**s to customize, and defines the **element** constraints or **extension**s in the FHIR **profile** to be created by the new model.
```

Comment:
Define the model specific CLASSES you need...

Reply:
Done

Reply:
It still says "Define the model specific Elements you need."